sw/airborne/firmwares/rotorcraft/navigation.h

```diff
@@ -128,6 +130,14 @@ extern bool nav_set_heading_current(void);
     dist2_to_wp = get_dist2_to_waypoint(_wp); \
   }
 
+#define NavGotoWaypointHeading(_wp) { \
```

Comment:
There is no good reason why this should be a macro instead of a static inline function...